.github/workflows/claude-auto.yml

```
@@ -0,0 +1,59 @@
+    types: [opened, reopened, synchronize]
+
+permissions:
+  contents: write
+  pull-requests: write
+  issues: write
+
+jobs:
+  generate-and-pr:
+    runs-on: ubuntu-latest
+    # APIキーが未設定の場合はスキップ
+    if: ${{ secrets.ANTHROPIC_API_KEY != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)